includes/MediaWiki/ReconciliationItem.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace MediaWiki\Extension\WikibaseReconcileEdit\MediaWiki;
+
+use Wikibase\DataModel\Entity\Item;
+
+/**
+ * Class that contains the item and revision found by the reconciliation service
+ * @see ReconciliationService
+ */
+class ReconciliationItem {
```

Comment:
Which would mean we would have to keep checking for type and cast it (or refactor all other things relying on this being an item). 

Do we see a use-case for anything other than items? I guess we could extend EntityRevision with some helper function that casts the item but that feels kinda awkward too considering how simple both classes are.

Reply:
We could even just do

```php
/** @var Item $item */
$item = $entityRevision->getEntity();
```

and call it a day, since PHP is a dynamic language…